### DISCUSSION TITLE
How to set a different dataloader for each gpu?

### DISCUSSION THREAD

**Discussion by rakhimovv** [Lightning-AI/pytorch-lightning#6827] [DDP / multi-GPU / multi-node]
For example, I have 4 object classes and want each of 4 gpus to work on its own unique class only. The story behind is that I have some gpu-memory-loaded preprocessing for each class label. Could you recommend, please, how one can do that?
